src/plugins/lua/ratelimit.lua

```diff
@@ -560,14 +560,11 @@ if opts then
     -- new way of setting limits
     fun.each(function(t, lim)
       if type(lim) == 'table' then
-        if type(lim[2]) == 'number' then
-          settings[t] = lim
-        elseif type(lim[2]) == 'string' then
-          settings[t] = lim
-          local plim = parse_string_limit(lim[2])
-          if plim then
-            settings[t][2] = plim
-          end
+        settings[t] = lim
+      elseif type(lim) == 'string' then
+        local plim = parse_string_limit(lim)
+        if plim then
+          settings[t] = {0, plim, 1}
```

Comment:
Erm, that doesn't something I've meant. Why 0? The limit of bucket should be the first value (before slash).